spec/wrap.md

```diff
@@ -0,0 +1,104 @@
+# WRAP Specification
+
+WRAP defines the rules for bi-directional communication between a Host and a Wasm module by defining the processes of invocation and subinvocation. It makes use of Host allocated Wasm shared memory to pass encoded buffers between the Host and the Wasm module.
+
+The Wasm module must define the following imports and exports:
+
+#### Exports
+- `_invoke: (bufferLen: u32) => u32`
```

Comment:
What were your reasons for removing the "_wrap_" prefix? I do like the look of this, just curious. Maybe have a section in the doc that covers the considerations & design choices made from 0.1 -> 0.2, like we did in the ABI PR.

Reply:
I'm torn, we can talk about it on the ACDC